### DISCUSSION TITLE
TwoSampleMRR package, multivariable occurrence always pops up for me to verify, but my access_token = NULL, still appears how to solve

### DISCUSSION THREAD

**Discussion by HMUWYH** [MRCIEU/TwoSampleMR#490] [Q&A]
api_status()
Using access token. For info on how this is used see logging_info()
Waiting for authentication in browser...
Press Esc/Ctrl + C to abort
Authentication complete.
错误: Could not authenticate via any gargle cred function
> exp<-mv_extract_exposures(
+     id_exposure=c("ukb-b-2862","ukb-b-3881","ukb-b-15926","ukb-b-1996"),
+     clump_r2 = 0.001,
+     clump_kb = 10000,
+     harmonise_strictness = 2,
+     access_token = ieugwasr::check_access_token(),
+     find_proxies = TRUE,
+     force_server = FALSE,
+     pval_threshold = 5e-08,
+     pop = "EUR"
+ )
Using access token. For info on how this is used see logging_info()
Waiting for authentication in browser...
Press Esc/Ctrl + C to abort
![f16ba5422d261e923b52e5601ff3cbe](https://github.com/MRCIEU/TwoSampleMR/assets/99111166/a1bba7a8-40de-4822-b921-74c17fedb064)

**Comment by ikram2779** [comment]
same issue

> get_access_token()
Using access token. For info on how this is used see logging_info()
Waiting for authentication in browser...
Press Esc/Ctrl + C to abort
Authentication complete.
Error: Could not authenticate via any gargle cred function

**Comment by Terry-Liu88** [comment]
Hey, I also encountered the same problem, did you solve it?
